src/poprox_storage/repositories/qualtrics_survey.py

```diff
@@ -96,3 +100,45 @@ def store_survey_response(self, response: QualtricsSurveyResponse) -> UUID | Non
             },
             constraint="uq_qualtrics_response_id",
         )
+
```

Comment:
I don't really like the return types on either of the following methods, but they were the best I could think of without making larger modifications to the existing domain concept classes. It seems like the ideal setup from the querying side would be for `QualtricsSurveyResponse` to have a `QualtricsSurveyInstance` as an attribute, but that may require some changes to the storing-side code.

Reply:
We should capture this possible refactor somewhere. I agree this feels screwy in a "model objects are at the wrong level" type of way. but I also don't know if this is the PR for that refactor :/